### ISSUE TITLE
torch.clamp_max clamp_min are not documented

### ISSUE THREAD

**Issue by ezyang** [pytorch/pytorch#37443]
Maybe they're not supposed to be in torch namespace?

**Comment by arbfay** [comment]
`torch.nn.functional.normalize` uses `clamp_min`, so any PR should take care to replace it by `clamp(min=eps)`

https://github.com/pytorch/pytorch/blob/df837d0384398f1fac1000189d62b05e7ff0240d/torch/nn/functional.py#L4269-L4273

**Comment by albertz** [comment]
Adding to this, how is `clamp_min` different from `maximum`? From the documentation, there shouldn't be a difference. But in practice, for example `torch.maximum(x, 2)` does not work, while `clamp_min(x, 2)` works.
